package.json

```diff
@@ -122,6 +122,7 @@
     "lint-no-cache": "node_modules/.bin/eslint --quiet --ext .ts,.tsx frontend/javascripts/",
     "pretty": "node_modules/.bin/prettier --write --config .prettierrc \"frontend/javascripts/**/*.ts\" \"frontend/javascripts/**/*.tsx\" \"tools/**/*.js\"",
     "pretty-backend": "sbt \";scalafmt; util/scalafmt; webknossosTracingstore/scalafmt; webknossosDatastore/scalafmt\"",
+    "lint-backend": "sbt \";scapegoat; scalafmtCheck; util/scalafmtCheck; webknossosTracingstore/scalafmtCheck; webknossosDatastore/scalafmtCheck\"",
```

Comment:
Did you use this locally? I tend to just run pretty-backend, which mutates the file.
Maybe this could also be used in the CI directly?

Reply:
It also runs `scapegoat`, which I guess is the linter? Perhaps the `scalafmt` doesn't belong in this call